cypress/support/commands.ts

```diff
@@ -0,0 +1,21 @@
+export {};
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace Cypress {
+    interface Chainable {
+      checkHCOSpec(spec: string, matchString: string, include: boolean): void;
+    }
+  }
+}
+
+Cypress.Commands.add('checkHCOSpec', (spec: string, matchString: string, include: boolean) => {
+  cy.exec(
+    `oc get -n kubevirt-hyperconverged hyperconverged kubevirt-hyperconverged -o jsonpath='{${spec}}'`,
+  ).then((result) => {
```

Comment:
In case of error the method will skip the `expect()` checks. We need fail the check in `catch()`.

Reply:
Again, the error handle is leaving to cypress, if the command failed to execute, qe need to fix the comand.